TCAT/Views/SummaryView.swift

```diff
@@ -128,7 +124,9 @@ class SummaryView: UIView {
             
             mainLabel.attributedText = attributedString
             
+            // Find time within label to place live indicator
             if let stringRect = mainLabel.boundingRect(of: departDirection.startTimeWithDelayDescription + " ") {
+                print("setting origin")
```

Comment:
is print statement needed? seemed that ur using it for debugging. if so, delete.